src/mscorlib/corefx/System/Globalization/CalendarData.cs

```diff
@@ -302,6 +302,65 @@ private void InitializeAbbreviatedEraNames(string localeName, CalendarId calenda
                     break;
             }
         }
+
+        internal static CalendarData GetCalendarData(CalendarId calendarId)
+        {
+            //
+            // Get a calendar.
+            // Unfortunately we depend on the locale in the OS, so we need a locale
```

Comment:
This is actually common code now.  I moved it from the PAL into just CalendarData.cs.  This same code is applicable to both Win32 and Unix.